examples/basic_cog.py

```diff
@@ -0,0 +1,101 @@
+import discord
+from discord.ext import commands
+import random
+
+description = '''An example bot to showcase the discord.ext.commands extension
+module.
+There are a number of utility commands being showcased here.
+The example uses cogs to organize the code
+'''
+
+# The intents chosen are the default intents which allow us to get all but presence and member intents.
+# We use the member intent for the `on_member_join` event.
+# For more information on intents read https://discordpy.readthedocs.io/en/latest/intents.html#intents-primer.
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot(command_prefix='?',
+                   description=description, intents=intents)
+
+
+# This defines a class that contains commands in a "Miscellaneous" category
+# The library calls this concept a Cog and they must inherit from `commands.Cog`
+# Cogs are useful for grouping commands and having them share state
+# This class will group commands in a "Misc" category in the default help command
+# The `name` keyword argument passed allows us to set the name of the Cog
+# If a name is not given then it will default to the name of the class
```

Comment:
```suggestion
# This defines a class that contains commands in a "Miscellaneous" category.
# The library calls this concept a Cog and they must inherit from `commands.Cog`.
# Cogs are useful for grouping commands and having them share state.
# This class will group commands in a "Misc" category in the default help command.
# The `name` keyword argument passed allows us to set the name of the Cog.
# If a name is not given then it will default to the name of the class.
```